### PR TITLE
Add Migration doc update for Boolean Attribute type setting

### PR DESCRIPTION
**Pull request by Shaaali**
## Purpose
Migration doc update for https://github.com/wso2/product-is/issues/14027

## Goals
Introduce changes related to the fix https://github.com/wso2/product-is/issues/13978

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/deploy/migrate/what-has-changed.md [hunk 1]
```diff
@@ -657,4 +657,36 @@ In WSO2 Identity Server 6.0.0, the Log4j version is upgraded to ```Log4j2```. Th
 ``` js
 [hazelcast]
 hazelcast.logging.type = log4j2
+```
+### Change In Data Type for Boolean Attributes From Schemas Endpoint
+
+`scim2/Schemas` endpoint now returns the attribute type of boolean attributes as `boolean`. Before this fix, the attribute type was returned as `string`.
```
Comment (ashendes) [inline en/docs/deploy/migrate/what-has-changed.md:663]: Lets not use the word 'fix' in docs. Better to use `change` or `update` instead.

Comment (ashendes) [reply en/docs/deploy/migrate/what-has-changed.md:663]: `Before this fix` -> `In previous versions of the Identity Server`

Comment (divyaamunugama) [inline en/docs/deploy/migrate/what-has-changed.md:663]: Let's stick to the same format that we have been using. (i.e., first, we mention the previous version, and then the update)
```suggestion
In previous versions of WSO2 Identity Server, the `scim2/Schemas` endpoint returned the attribute type of boolean attributes as `string`. 

In Identity Server 6.0.0, the attribute type returned for boolean attributes of the `scim2/Schemas` endpoint is `boolean`.
```

File: en/docs/deploy/migrate/what-has-changed.md [hunk 2]
```diff
@@ -500,6 +500,36 @@ If this behavior is not required, it can be disabled using the following configu
 [scim2]
 remove_duplicate_users_in_users_response = true 
 ```
+
+#### Change In Data Type for Boolean Attributes from the Schemas Endpoint
```
Comment (divyaamunugama) [inline en/docs/deploy/migrate/what-has-changed.md:504]:
```suggestion
#### Data type change for boolean attributes from the Schemas endpoint
```

**Review by divyaamunugama — APPROVED** [review]

**Pull request merged**